wire-ios-system/Source/ZMSLog.swift

```diff
@@ -376,7 +376,7 @@ extension ZMSLog {
 
         do {
             let data = Data(string.utf8)
-            try updatingHandle?.wr_write(data)
+            try updatingHandle?.write(contentsOf: data)
```

Comment:
See deprecation from iOS 13 and later: https://developer.apple.com/documentation/foundation/nsfilehandle/1410936-writedata?language=objc